update.sh

```diff
@@ -36,6 +36,7 @@ nc-zram
 SSH
 fail2ban
 NFS
+metrics
```

Comment:
Unfortunately not, because the prometheus exporter requires direct (host) system access. I'll add instructions on how to set it up separately from ncp (on the host) later.
I could evaluate what container privileges are required, but it would be broken for all users that are not aware of those

Reply:
no worries then